posts/blazing-fast-ci-with-microvms.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Blazing fast CI with MicroVMs
```

Comment:
There's some kind of "og URL" that we can include in posts like this where it's copied from another website.

I think we'll need that in place before we merge.

Like:

`<link rel="canonical" href="http://﻿﻿﻿silvernailwebdesign.com/reposting-articles"/﻿﻿﻿>﻿﻿﻿﻿`

https://silvernailwebdesign.com/reposting-articles/


```yaml
canonical: https://blog.alexellis.io/blazing-fast-ci-with-microvms/
```

Reply:
Setting `canonical` in the post frontmatter is now supported.